app/views/studies/_expression_file_info_fields.html.erb

```diff
@@ -27,20 +27,20 @@
     </div>
     <div class="col-lg-2">
       <%= f.label :biosample_input_type, label_with_asterisk('Biosample Input Type') %><br />
-      <%= f.select :biosample_input_type, options_for_select(ExpressionFileInfo::BIOSAMPLE_INPUT_TYPE_VALUES, f.object.biosample_input_type),
-                   {}, class: 'form-control' %>
+      <%= f.select :biosample_input_type, options_for_select(ExpressionFileInfo::BIOSAMPLE_INPUT_TYPE_VALUES, f.object.study_file.name.present? ? f.object.biosample_input_type : 'Whole cell'),
```

Comment:
While this will have the intended outcome of specifying a default, it also means that the "default" logic lives in the form, and not in the data model itself.  Since we are actively considering refactoring the data upload/sync UX, this kind of compartmentalized logic will lead to regressions moving forward.

I would instead posit that it would be better to specify the defaults at the model level, rather than the UI level:

```
class ExpressionFileInfo

  ...
  field :library_preparation_protocol, type: String, default: 'Transcriptomic: unbiased'
  field :biosample_input_type, type: String, default: 'Whole cell'
  field :modality, type: String
  ...
```

This means that if we execute `study_file.build_expression_file_info`, the resulting document already has the default values set, and the form logic for all of these fields doesn't need to change (except for adding the "Select one" prompts as you have done).

Reply:
I can see this going both ways.  Agree with your point, Jon about the model being a more central place to find defaults.  But having a default at the model level is dangerous because that can lead it to being accidentally set when we actually had no information provided, and would rather the API return an error.   Having a default value means we will have to be more careful in our controller code at making sure we're preserving nils/undefined that got passed in

Reply:
> But having a default at the model level is dangerous because that can lead it to being accidentally set when we actually had no information provided

Fair point, but we won't have this scenario as all of these values have been required all along, so we don't have any documents where data hasn't been set.  This will only impact new files, and we want to prompt with these new defaults.  I think my previous points still hold.